src/similarity.rs

```diff
@@ -445,13 +493,31 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_embedding_similarity() {
+        let f = File::open("testdata/similarity.bin").unwrap();
+        let mut reader = BufReader::new(f);
+        let embeddings = Embeddings::read_word2vec_binary(&mut reader).unwrap();
+        let embedding = embeddings.embedding("Berlin").unwrap();
+        let result = embeddings.embedding_similarity(embedding.as_view(), 10);
+        assert!(result.is_some());
+        let mut result = result.unwrap().into_iter();
+        assert_eq!(10, result.len());
+        assert_eq!(result.next().unwrap().word, "Berlin");
+        println!("{:?}", result);
```

Comment:
Stray `println!`. I know it's also in the other test, but we shouldn't introduce it in this test as well ;).